src/backend/trace.rs

```diff
@@ -410,12 +410,12 @@ impl TraceData {
                 hop.total_time += dur;
                 // Before last is set use it to calc jitter
                 let last_ms = hop.last_ms().unwrap_or_default();
-                let jitter_ms = (last_ms - dur_ms).abs();
+                let jitter_ms = (dur_ms - last_ms).abs();
                 let jitter_dur = Duration::from_secs_f64(jitter_ms / 1000_f64);
                 hop.jitter = hop.last.and(Some(jitter_dur));
                 hop.javg += (jitter_ms - hop.javg) / hop.total_recv as f64;
                 // algorithm is from rfc1889, A.8 or rfc3550
-                hop.jinta += jitter_ms - ((hop.jinta + 8.0) / 16.0);
+                hop.jinta += jitter_ms.max(0.5) - ((hop.jinta + 8.0) / 16.0);
```

Comment:
@trkelly23 can you help me understand the math here?

Reply:
I found during extreme low-latency testing (jitter_ms .lt .5 millis) the jinta equation returns negative values.  I added a floor of .5 millis as a fix for the bug.
From mtr the jinta equation is jinta = jitter - ((nh->jinta + 8) >> 4);
https://github.com/traviscross/mtr/blob/master/ui/net.c#L304

I update ours to divide by 16.0 which is same as >>4.  I preferred using floats to avoid having to force a conversion to float of the result using "as f64", it a little cleaner this way.

Reply:
I've moved this to a separate `fix` commit.